parquet-tools/src/main/java/parquet/tools/command/ShowSchemaCommand.java

```diff
@@ -68,7 +71,21 @@ public void execute(CommandLine options) throws Exception {
     String input = args[0];
 
     Configuration conf = new Configuration();
-    ParquetMetadata metaData = ParquetFileReader.readFooter(conf, new Path(input));
+    ParquetMetadata metaData;
+
+    Path path = new Path(input);
+    FileSystem fs = path.getFileSystem(conf);
+    Path file;
+    if (fs.isDirectory(path)) {
+      FileStatus[] statuses = fs.listStatus(path, HiddenFileFilter.INSTANCE);
+      if (statuses.length == 0) {
+        throw new RuntimeException("Directory " + path.toString() + " is empty");
+      }
+      file = statuses[0].getPath();
+    } else {
+      file = path;
+    }
+    metaData = ParquetFileReader.readFooter(conf, file);
```

Comment:
Should we take schema evolution into account here? I.e., show the merged schemas of all part-files.